### PR TITLE
IIS 'first experience' tutorial

### LINKED ISSUE #13309: Create an IIS 'first experience' tutorial
I can't get past "Deploy the App".

I ended up on this page after following the Tutorials to create a sample .NET Core 2.2 Web API, which I was successful!  The "next steps" were to deploy the app.

So... I'm trying to deploy that app to an IIS box running ASP.net Core 2.2 (as per the first half of this guideline).

I'm attempting to follow the section "Deploy the app", but I'm totally lost.  There is no context to any of this.  This article is nothing but a bunch of words.  Who is the intended audience?  Do you plan to write something a little more concrete, and easy-to-follow?

IMHO, "installing" and "deploying" should be two separate articles, and they should tie in with the other tutorials.

I've been trying to nail up a sample .NET core site for weeks now.  Where can I find a solid example to follow?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ec484002-18c1-6a0d-00de-820d56f038d3
* Version Independent ID: 7903ce53-2cbb-f083-7479-c8abc30211a0
* Content: [Host ASP.NET Core on Windows with IIS](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-2.2#deploy-the-app)
* Content Source: [aspnetcore/host-and-deploy/iis/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/iis/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #13309

[Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/publish-to-iis?view=aspnetcore-2.1&branch=pr-en-us-13593)

* A "build-your-first"-flavor IIS experience for ASP.NET Core ... 😃 *The Happy Path*:tm: 😃
* Needed because the IIS Overview topic has become a 😵 **_13 Chapter BOOK_** 😵 of info for IIS deployments. Readers are having trouble focusing on the three main sections of the topic that get an ASP.NET Core app on IIS with default config.
* There's a lot of trimming of advanced concepts here. I'm linking out and warning readers ... striking a balance. See if these choices are sane. Add content? Trim content further? :ear::ear::ear:

Thanks @raydlevel5! :guitar:

**IGNORE those commit build warnings. Those are *false positives* on the tabs.**

### CODE REVIEW

**Comment by shirhatti** [comment]
Looks good on a cursory look. A few things that were missing that should be linked to or included:
- How to create a registry hive for data protection
- How to ACL your directory when creating the IIS site

**Comment by guardrex** [comment]
@shirhatti I *guessed* that this would be the most basic level of deployment possible ... just an *up-and-running* experience. I left those to the IIS topic, and I have that IMPORTANT note about config and security at the top with the cross-link. I say explicitly not to attempt hosting a production app without consuming the guidance of the main IIS topic. Perhaps, I need to strengthen that lingo. I'll edit this again on Thursday.

**_Side note on our tutorial guidance_**: We aren't allowed to cross-link much in the middle of this type of topic. We generally must either add the content or rely upon the *Additional resources* section. It's a docs guideline thing.

**Comment by guardrex** [comment]
@shirhatti I think I was successful in 🐄 *beefing up* 🐄 that *Prerequisites* section. It now specifically surfaces and cross-links those two items that you listed.

**Comment by guardrex** [comment]
@Rick-Anderson Suggest we go ahead with our review. I addressed @shirhatti feedback on the last commit.

File: aspnetcore/tutorials/publish-to-iis.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/31/2019
+uid: tutorials/publish-to-iis
+---
+# Publish an ASP.NET Core app to IIS
+
+By [Luke Latham](https://github.com/guardrex)
+
+This tutorial shows you how to host an ASP.NET Core app on an IIS server.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/publish-to-iis.md]: This tutorial shows how to host an ASP.NET Core app on an IIS server.

File: aspnetcore/tutorials/publish-to-iis.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+# [Visual Studio](#tab/visual-studio)
+
+1. Right-click on the project in **Solution Explorer** and select **Publish**.
+1. In the **Pick a publish target** dialog, select the **Folder** publish option.
+1. Set the **Folder or File Share** path.
+   * If you created a folder for the IIS site that's available on the development machine as a network share, provide the path to the share. The current user must have write access to publish to the share.
+   * If you're unable to deploy directly to the IIS site folder on the IIS server, publish to a folder on removeable media and physically move the published app to the IIS site folder on the server, which is the site's **Physical path** in IIS Manager. Move the contents of the *bin/Release/{TARGET FRAMEWORK}/publish* folder to the IIS site folder on the server, which is the site's **Physical path** in IIS Manager.
+
+# [Visual Studio Code / .NET Core CLI](#tab/visual-studio-code+netcore-cli)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/publish-to-iis.md]: I think you should just use VSC or .NET Core CLI, probably CLI
This isn't an approved tab
cc @scottaddie  @tdykstra

Comment (guardrex) [reply aspnetcore/tutorials/publish-to-iis.md]: Prior guidance was to combine tabs. @scottaddie said that that the error is a false positive. Currently, this tab is used in six topics.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/publish-to-iis.md]: We've got approval to combine VSC/Mac. You can't combine any two tabs. Go with one or the other, they're both not required. I recommend CLI. Folks will figure this out.

Comment (guardrex) [reply aspnetcore/tutorials/publish-to-iis.md]: Sure thing ... noting that this is a change in my guidance. I specifically asked about this tab and that was the answer that I was given. I'll fix it now.

**Comment by guardrex** [comment]
One more to fix ... a bookmark. I'll merge this when I get that sorted out.

**Comment by guardrex** [comment]
Those must be inconsistencies on the branch that were fixed on other PRs. We're good here.

**Pull request merged**